src/Dofs/apply_analytical.jl

```diff
@@ -56,7 +56,8 @@ function _apply_analytical!(
     ref_points = reference_coordinates(ip_fun)
     dummy_weights = zeros(length(ref_points))
     qr = QuadratureRule{dim, RefShape}(dummy_weights, ref_points)
-    cv = CellScalarValues(qr, ip_fun, ip_geo)
+    # Note: Passing ip_geo as the function interpolation here, it is just a dummy.
+    cv = CellScalarValues(qr, ip_geo, ip_geo)
```

Comment:
This should be constructed via some helper dispatch to extract the scalar-valued interpolation in case of vectorization, right? (e.g. when we use Serendipity for the solution and Lagrange for the geometry)

Reply:
Doesn't really matter, it is only used for `spatial_coordinate`, could be an [empty interpolation](https://dealii.org/developer/doxygen/deal.II/classFE__Nothing.html#a69940158d653b46222943f09c6b25a4c).

Reply:
Ah, I see. So we do not actually need a full CellScalarValues here but something cheaper could be sufficient. Separate issue then.

Reply:
Could extract
```
struct Mapping
    M
    dMdξ
end
end
```
that can be used standalone.